src/legacy/core_plugins/kibana/public/visualize/editor/editor.html

```diff
@@ -33,6 +33,7 @@
     show-query-bar is set to "true".
   -->
   <kbn-top-nav-v2 
+    ng-show="chrome.getVisible()"
```

Comment:
Is this meant to be updated in this PR?

Reply:
Nope, not sure why I thought I needed this.